.snyk

```diff
@@ -3,13 +3,13 @@ ignore:
   "snyk:lic:golang:github.com:hashicorp:vault:sdk:MPL-2.0":
     - "*":
         reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
-  "snyk:lic:golang:github.com:hashicorp:vault:MPL-2.0":
```

Comment:
No longer needed b/c the `github.com/hashicorp/vault` dependency is no longer used.